src/style.css

```diff
@@ -0,0 +1,48 @@
+body {
+  background-color: black;
+  display: flex;
+  flex-direction: column;
+}
+
+:root {
+  --yellow: #ffa500;
+  --red: #e75551;
+  --blue: #308dc0;
+  --text-background: #ddd;
+}
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Try to avoid `*` selector. In these tasks it doesn’t influence performance, but in bigger projects, it can load browser